### ISSUE TITLE
Solve Xor

### ISSUE THREAD

**Issue by cswinter** [entity-neural-network/incubator#102]

**Comment by cswinter** [comment]
Solved on eef5ced with the command:

```
poetry run python enn_ppo/enn_ppo/train.py --total-timesteps=50000 --track --gym-id=Xor --num-envs=512 --max-log-frequency=1000 --n-layer=2 --num-steps=1
```

Proof: https://wandb.ai/entity-neural-network/enn-ppo/runs/3kfwpk9w?workspace=user-cswinter

**Comment by cswinter** [comment]
This works more robustly:

```
poetry run python enn_ppo/enn_ppo/train.py --total-timesteps=500000 --track --gym-id=Xor --num-envs=512 --n-layer=2 --num-steps=1 --ent-coef=0.1
```
